renderer/types/Minor.ts

```diff
@@ -9,3 +9,11 @@ export type MinorParser = {
     };
   }[];
 };
+
+export type Metrics = {
```

Comment:
I think @vird already placed this type were it can be reused by renderer as well as main thread?